scripts/generate_dump

```diff
@@ -460,6 +460,11 @@ main() {
         fi
     done
 
+    # run 'hw-management-generate-dump.sh' script and save the result file
+    /usr/bin/hw-management-generate-dump.sh    
+    save_file "/tmp/hw-mgmt-dump*" "hw-mgmt" false
+    rm -f /tmp/hw-mgmt-dump*
```

Comment:
Should we surround this with an `if [ "$asic" = "mellanox" ]` condition?

Reply:
indeed. sorry for missing it. @shlomibitton please handle

Reply:
added